apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -170,7 +172,10 @@ groups() ->
       ]},
      {initiator_is_ga, [sequence], ga_sequence()},
      {responder_is_ga, [sequence], ga_sequence()},
-     {both_are_ga, [sequence], ga_sequence()}
+     {both_are_ga, [sequence], ga_sequence()},
+     {client_reconnect, [sequence],
+      [ client_reconnect_initiator
+      , client_reconnect_responder ]}
```

Comment:
There is no test case for an unauthorized user trying to reconnect to the FSM. We don't check what happens when someone tries to reconnect while the FSM has a connected client.

Reply:
This is now checked.